tsl/src/continuous_aggs/materialize.c

```diff
@@ -665,12 +791,22 @@ scan_update_invalidation_threshold(TupleInfo *ti, void *data)
 	Form_continuous_aggs_invalidation_threshold form =
 		(Form_continuous_aggs_invalidation_threshold) GETSTRUCT(new_tuple);
 	if (form->watermark > new_threshold)
-		elog(ERROR, "Internal Error: new invalidation threshold must not be less than the old one");
-	form->watermark = new_threshold;
-	ts_catalog_update(ti->scanrel, new_tuple);
+		elog(INFO,
```

Comment:
why is this INFO log still here? If we want to keep it for debugging, the message should be more informative.

Reply:
modified the message.